crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -15,10 +15,16 @@ use shared::command::CommandExt;
 
 mod command;
 
+/// The offset of the sierra statement within the bytecode.
+type CodeOffset = usize;
+
+/// The index of the sierra statement in the instructions vector.
+type InstructionIdx = usize;
```

Comment:
I don't think this comment is true.

Reply:
Isn't intruction vector a vector of casm instructions?

Reply:
It is ambiguous, should be specified imo that we are talking about sierra instructions

Reply:
Again, i copied the comments